src/Bridges/Constraint/bridges/functionize.jl

```diff
@@ -33,6 +33,15 @@ function MOI.get(
     return MOI.get(model, attr, bridge.constraint)
 end
 
+function MOI.get(
+    model::MOI.ModelLike,
+    ::MOI.CanonicalConstraintFunction,
+    bridge::AbstractFunctionConversionBridge,
+)
+    f = MOI.get(model, MOI.ConstraintFunction(), bridge)
+    return MOI.Utilities.canonical(f)
```

Comment:
Shouldn't it convert like the initial methods that you are replacing ?

Reply:
No. It now calls `ConstraintFunction`, which must be the same type, so no need for `convert` here.

The two implementations below use `convert`, but it should also hold for any other subtype of `AbstractFunctionConversionBridge`.